src/supremacy/player.py

```diff
@@ -191,6 +191,7 @@ def rip(self):
 
     def dump_map(self):
         im = Image.fromarray((self.game_map.array.astype(np.uint8) + 1) * 127)
```

Comment:
How about just using
```suggestion
        im = Image.fromarray(np.flipud((self.game_map.array.astype(np.uint8) + 1) * 127))
```
?

Reply:
To be honest, I do not know either numpy or pillow good enough to have an educated opinion about possible performance difference of the two approaches. Whichever solution you prefer is fine for me.